testbed/tests/trace_test.go

```diff
@@ -115,6 +115,108 @@ func TestTrace10kSPS(t *testing.T) {
 	}
 }
 
+func TestTrace10kSPSCompressed(t *testing.T) {
```

Comment:
What about we add the gzip to the already existing `TestTrace10kSPS` to avoid running the non-gzip twice?

Reply:
Done.